auth-web/src/router.ts

```diff
@@ -109,7 +111,7 @@ export function getRoutes (): RouteConfig[] {
     { path: '/returnpayment/:paymentId/transaction/:transactionId', name: 'returnpayment', component: PaymentReturnView, props: mapReturnPayVars, meta: { requiresAuth: false, requiresProfile: true } },
     { path: '/searchbusiness', name: 'searchbusiness', component: SearchBusinessView, props: true, meta: { requiresAuth: true, allowedRoles: [Role.Staff] } },
     { path: '/unauthorized', name: 'unauthorized', component: UnauthorizedView, props: true, meta: { requiresAuth: false } },
-    { path: '/unauthorizedtermsdecline', name: 'unauthorizedtermsdecline', component: UnauthorizedView, props: true, meta: { requiresAuth: true } },
+    { path: '/unauthorizedtermsdecline', name: 'unauthorizedtermsdecline', component: TermsOfServiceDeclineView, props: true, meta: { requiresAuth: true } },
```

Comment:
I hope this view is not used anywhere else

Reply:
nope, this view is only for the terms decline view, earlier I reused Unauthorized page conditionally for this, but it seems like there are a lot of different conditions needed in that, thats why moved to a new view page